report/components/core/labels/AuthorLabel.tsx

```diff
@@ -18,21 +18,17 @@ const _AuthorLabel = ({ index }: LabelProps) => {
 
     // add discriminator in Discord
     if (db.config.platform === "discord") {
-        let n = author.n;
-        let discr = n.split("#").pop();
+        const discr = author.n.split("#").pop();
 
         // only keep if it's 4 chars (and not a deleted ID)
         if (discr && discr.length === 4) {
-            discr = parseInt(discr).toString();
-            n = n.slice(0, -5);
-        } else discr = undefined;
-
-        name = (
-            <>
-                {n}
-                {discr && <span className="Label__discriminator">#{`${isDemo ? 0 : discr}`.padStart(4, "0")}</span>}
```

Comment:
The new version does not have the `padStart` when the discriminator is less than 1000

Reply:
What do you mean? It works for me, for example on the Demo the #0000 shows correctly, and I've tried my own reports where I didn't see anything like #900 for #0900

Reply:
Oh just realised that the code you referenced was the old code, I already removed it 😅As I say though, it works for me, because the discr is being kept as a string rather than being parsed to an integer first meaning the padding is already there

Reply:
My bad, `message.author.discriminator` is already padded